src/components/Calculator.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './Calculator.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { output: 0 };
+  }
+
+  render() {
+    const { output } = this.state;
+    return (
+      <div>
+        <table className="table table-bordered">
+          <tbody>
+            <tr className="output">
+              <td colSpan="4">{output}</td>
+            </tr>
+            <tr>
+              <td>AC</td>
+              <td>+/-</td>
+              <td>%</td>
+              <td className="orange-colour">÷</td>
+            </tr>
+            <tr>
+              <td>7</td>
+              <td>8</td>
+              <td>9</td>
+              <td className="orange-colour">x</td>
+            </tr>
+            <tr>
+              <td>4</td>
+              <td>5</td>
+              <td>6</td>
+              <td className="orange-colour">-</td>
+            </tr>
+            <tr>
+              <td>1</td>
+              <td>2</td>
+              <td>3</td>
+              <td className="orange-colour">+</td>
```

Comment:
**Optional Changes**
Good effort using tables to create the different buttons.
- [ ] I will advice that you store the different buttons inside any `array` and then use `array.map` to dynamically display then in the `DOM`. This will allow us to stick to the `DRY` principle and it will prove handy in the proceeding sections.